README.md

```diff
@@ -99,6 +99,10 @@ services:
 
 By default, Muse limits the total cache size to around 2 GB. If you want to change this, set the environment variable `CACHE_LIMIT`. For example, `CACHE_LIMIT=512MB` or `CACHE_LIMIT=10GB`.
 
+### SponsorBlock
+
+Muse can skip non-music segments at the beginning or end of a Youtube music video (Using [SponsorBlock](https://sponsor.ajay.app/)). It is disabled by default. If you want to enable it, set the environment variable `ENABLE_SPONSORBLOCK=true` or uncomment it in your .env.
```

Comment:
Attribution in compliance with https://gist.github.com/ajayyy/4b27dfc66e33941a45aeaadccb51de71